menpo/landmark/test/landmark_test.py

```diff
@@ -118,12 +118,12 @@ def test_LandmarkManager_group_order():
     man = LandmarkManager()
     man['test_set'] = pcloud.copy()
     man['abc'] = pcloud.copy()
-    man['def'] = pcloud.copy()
```

Comment:
@patricksnape (I'll be honest, before trying that I actually thought the first two might succeed, as I'd see the latter example work before. But, (obviously, in retrospect) `__builtin__ != keyword`)